src/pages/contact/Contact.tsx

```diff
@@ -0,0 +1,165 @@
+import { Trans, t } from '@lingui/macro'
+import { Divider, Button, Form, Input, Select, Row, Col } from 'antd'
+import Image from 'next/image'
+import { useState } from 'react'
+import apple from '/public/assets/apple-ol.webp'
+import useMobile from 'hooks/Mobile'
+import { createContactMessage } from 'lib/api/discord'
+
+export default function Contact() {
+  const [contactPlaceholder, setContactPlaceholder] = useState<string>(
+    'banny@juicebox.money',
+  )
+  const [loading, setLoading] = useState<boolean>(false)
+  const [success, setSuccess] = useState<boolean>(false)
+  const [error, setError] = useState<boolean>(false)
+  const [submitDisabled, setSubmitDisabled] = useState<boolean>(true)
+  const isMobile = useMobile()
+
+  const [form] = Form.useForm()
+
+  const onFormSubmit = async () => {
+    setError(false)
+    setSuccess(false)
+
+    try {
+      setLoading(true)
+      await createContactMessage(
+        form.getFieldValue('name'),
+        form.getFieldValue('contact'),
+        form.getFieldValue('contactPlatform'),
+        form.getFieldValue('subject'),
+        form.getFieldValue('message'),
+      )
+      setSuccess(true)
+    } catch (e) {
+      setError(true)
+    } finally {
+      setLoading(false)
+      setSubmitDisabled(true)
+    }
+  }
+
+  const handleSelect = (option: string) => {
+    const examples = new Map<string, string>([
+      ['email', 'banny@juicebox.money'],
+      ['discord', 'banny#4200'],
+      ['twitter', '@banny'],
+      ['telegram', '@banny'],
+    ])
+
+    setContactPlaceholder(examples.get(option) ?? 'Your contact')
+  }
```

Comment:
**praise** nice idea!

Reply:
will these be the placeholders for the contact form @filipvvv ?

Reply:
@strath-m yes — when a user selects a platform to contact them on, a different placeholder appears in the format appropriate to that platform.